specifyweb/specify/models_utils/lock_tables.py

```diff
@@ -13,8 +12,59 @@ def lock_tables(*tables):
         yield
     else:
         try:
+            # NOTE: Should not do within a transaction.atomic() block
```

Comment:
(optional)

Can you provide a reference here to #6490, https://github.com/specify/specify7/issues/6490#issuecomment-3020675840, https://github.com/specify/specify7/issues/6490#issuecomment-3340619060, and/or this PR?